core/src/main/scala/org/http4s/parser/SimpleHeaders.scala

```diff
@@ -57,7 +57,7 @@ private[parser] trait SimpleHeaders {
   }
 
   def CONTENT_LENGTH(value: String): ParseResult[`Content-Length`] = new Http4sHeaderParser[`Content-Length`](value) {
-    def entry = rule { Digits ~ EOL ~> {s: String => `Content-Length`(s.toLong)} }
+    def entry = rule { Digits ~ EOL ~> {s: String => `Content-Length`.unsafeFromLong(s.toLong)} }
```

Comment:
Yeah, I wouldn't worry about it: worst case is numeric overflow on the `.toLong` and that won't yield a negative Long.